scrapy/core/downloader/handlers/aiohttp.py

```diff
@@ -0,0 +1,85 @@
+import asyncio
+from typing import Self
```

Comment:
Please import this from typing_extensions like we do in other files (and under `if TYPE_CHECKING`, and with `from __future__ import annotations`)